modules/acquia_cms_common/acquia_cms_common.info.yml

```diff
@@ -44,3 +44,4 @@ dependencies:
   - drupal:taxonomy
   - drupal:username_enumeration_prevention
   - drupal:workbench_email
+  - drupal:syslog
```

Comment:
@vishalkhode1 need your feedback here. As per my understanding, the syslog module should be enabled by default. So it should be enabled on site install? If yes, then hence the module is mentioned here. Else the file changes in `ToggleMofuleService.php` file below should help achieve the required functionality.

Thanks

Reply:
@panshulK This looks correct to me though, but for now let's enable only for the new customers who are using that toggle service. Though, I'm unsure. @fiasco can you suggest your opinion on this ?